icu_base.py

```diff
@@ -117,16 +121,17 @@ def build(self):
 
                     self.run(self._build_config_cmd, win_bash=tools.os_info.is_windows)
                     if self.options.get_safe("silent"):
```

Comment:
original code seems like it's double checking - can get_safe() return something that evaluates as a boolean different from ```self.options.slient```?  I would think not...

Reply:
`get_safe()` returns the value or `None` if it's not defined. `None` has falsy value.

The problem with the old code is that if `silent` was `False`, it would use the `else:` clause and set `--silent` anyway. Not helpful for debugging.

`icu_base.py` is subclassed by one build that has a `silent` option, and one that does not, which is why it's using `get_safe()`